build/build-image/cross/Dockerfile

```diff
@@ -32,8 +43,12 @@ ENV KUBE_CROSSPLATFORMS \
   darwin/amd64 darwin/386 \
   windows/amd64 windows/386
 
+CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
```

Comment:
why QEMUARCH is not an ARG?

Reply:
@javier-b-perez please see the Makefile, we replace `QEMUARCH` using `sed`

Reply:
is there a reason we still do that? when I dug into this last time it seemed like it was just legacy to do this vs ARG